docs/ppp/iteration-4-08112022.md

```diff
@@ -0,0 +1,25 @@
+### Plans 📆
+
+- Fund the on-chain wallet - https://github.com/itchysats/10101/issues/113
+- Setup and build the maker lightning node boilerplate - https://github.com/itchysats/10101/issues/89
+- Open a channel between taker and maker - https://github.com/itchysats/10101/issues/10
+- Persisting custom outputs - https://github.com/itchysats/rust-lightning/issues/2
+
```

Comment:
quite important plan missing: get the rust-lightning working without our 10101 app